examples/src/main/java/com/mapbox/navigation/examples/core/MultipleStopsActivity.kt

```diff
@@ -161,6 +165,18 @@ class MultipleStopsActivity : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
+    val arrivalObserver = object : ArrivalController {
+        val arrivalOptions = ArrivalOptions.Builder()
+            .arriveInSeconds(30.0)
+            .build()
+        override fun arrivalOptions(): ArrivalOptions = arrivalOptions
+
+        override fun onArrival(routeLegProgress: RouteLegProgress): Boolean {
+            Timber.i("arrival_debug distanceRemaining ${routeLegProgress.distanceRemaining()}")
```

Comment:
debugging this because it goes negative and stays really negative for some reason